core/object/object.h

```diff
@@ -87,6 +87,7 @@ enum PropertyHint {
 	PROPERTY_HINT_PASSWORD,
 	PROPERTY_HINT_LAYERS_AVOIDANCE,
 	PROPERTY_HINT_DICTIONARY_TYPE,
+	PROPERTY_HINT_INPUT_NAME,
 	PROPERTY_HINT_TOOL_BUTTON,
```

Comment:
```suggestion
	PROPERTY_HINT_TOOL_BUTTON,
	PROPERTY_HINT_INPUT_NAME,
```

Of course, until 4.4 we have the right to add constants in any order as long as it does not break compatibility with 4.3, but I would prefer to keep the chronological order.